helm_chart/templates/configmap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: {{ .Release.Name }}-configmap
+    namespace: {{ .Values.namespace }}
```

Comment:
No need to add namespace like this, it will be deployed in the namespace that you specify via terraform or helm binary.
Other option if you want to put it is to take it from Release Namespace instead of values.